### ISSUE TITLE
training grpo: loss is not 0, but log_pro is always the same with old_log_pro? Does the model not update?

### ISSUE THREAD

**Issue by zyf8818** [volcengine/verl#2501]
I have trained the model with grpo, but negative_approx_kl = log_prob - old_log_prob  ratio = torch.exp(negative_approx_kl) the ratio is always 1.0

> 

<img width="297" height="663" alt="Image" src="https://github.com/user-attachments/assets/044f14a7-ec02-4aff-9442-74bc22065c93" />

yellow is the 0-th mini_batches, I think it should be 1.0
read is the 1-th mini_batches, I think it should not be 1.0, because the model have been updated.

**Comment by yeahhh999** [comment]
Hi, I’ve encountered the same issue. Is this normal? Do you have any solutions?

**Comment by Kirrito-k423** [comment]
### 中文
定时清理检查结论：这个 issue 最近一次非当前运行账号的公开回复或引用是 `2025-08-28T06:24:08Z`，截至本轮运行时间 `2026-07-10T01:03:01Z` 已经超过 6 个月。检查来源包括 issue comments API `https://api.github.com/repos/verl-project/verl/issues/2501/comments` 和 timeline API `https://api.github.com/repos/verl-project/verl/issues/2501/timeline`；最近证据：https://github.com/verl-project/verl/issues/2501#issuecomment-3232086741。

因此按社区 issue 定时清理标准关闭。若作者或社区成员仍能提供当前版本的复现、日志、PR 关联或新的需求背景，欢迎继续评论请求重开，或创建一个带有最新证据的新 issue。

### English
Scheduled cleanup check: the latest public reply or reference from an account other than the current automation account was `2025-08-28T06:24:08Z`, which is more than 6 months before this run time `2026-07-10T01:03:01Z`. The checked public sources were the issue comments API `https://api.github.com/repos/verl-project/verl/issues/2501/comments` and the timeline API `https://api.github.com/repos/verl-project/verl/issues/2501/timeline`; latest evidence: https://github.com/verl-project/verl/issues/2501#issuecomment-3232086741.

Closing this under the scheduled issue cleanup policy. If the author or community can still provide a current reproduction, logs, PR reference, or updated requirement context, please comment to request reopening or open a new issue with fresh evidence.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
